python-flask/app/api/routes.py

```diff
@@ -123,3 +123,9 @@ def follow(username):
                 db.session.commit()
             return "", 204
 
+    elif request.method == "GET":
+        no_followers = request.args.get("no", type=int, default=100)
```

Comment:
Nice! Maybe we can add in the requirements to limit  the default as 100. Correct if I'm wrong but it is not there yet.